doc/getting-started.md

```diff
@@ -35,6 +41,36 @@ This article is a guide for installing the base Uno.Toolkit library, additional
         </ResourceDictionary.MergedDictionaries>
     </ResourceDictionary>
     ```
+### [**dotnet new**](#tab/dotnet-new)
+
+Option 1:- When creating a new project using the `dotnet new` templates, make sure  to include the [`-toolkit` option](xref:Uno.GettingStarted.UsingWizard.Features#toolkit) in the `dotnet new unoapp` command.     
+    
+    `dotnet new unoapp -preset blank -o MyApp -toolkit` 
+
+Option 2:- To incorporate the Uno Toolkit into an existing application, you must add the [Uno.Toolkit.WinUI NuGet package](https://www.nuget.org/packages/Uno.Toolkit.WinUI) to the **App Code Library** project as well as the platform heads. An easy way to do this is through the command line. Run the following command within the directory of each `.csproj`:       
+    
+    `dotnet add package Uno.Toolkit.WinUI --version [latest version number]. 
+
+ Make sure to replace `[latest version number] `with the actual latest NuGet version number.
+ 
+ Add the resources to `AppResources.xaml`:
+
+    ```xml
+    <ResourceDictionary>
+        <ResourceDictionary.MergedDictionaries>
+
+            <!-- Load Uno.Toolkit.UI resources -->
+            <ToolkitResources xmlns="using:Uno.Toolkit.UI" />
+
+            <!-- Load custom application resources -->
+            <!-- ... -->
+
+        </ResourceDictionary.MergedDictionaries>
+    </ResourceDictionary>
+    ```
+
+***
+
 
 #### Previous Installation Method
```

Comment:
i would argue that "previous installation method" on the same heading level as "installation"
if you have chosen heading 2 `##` for installation, then do the same for other.
```suggestion
## Previous Installation Method
```